.travis.yml

```diff
@@ -100,5 +100,18 @@ gemfile:
   - gemfiles/rails_4.2.5.1.gemfile
   - gemfiles/rails_4.2.5.2.gemfile
   - gemfiles/rails_4.2.6.gemfile
+  - gemfiles/rails_4.2.7.gemfile
+  - gemfiles/rails_4.2.7.1.gemfile
+
+matrix:
+  include:
+    - rvm: 2.2.2
+      gemfile: gemfiles/rails_5.0.1.gemfile
+      gemfile: gemfiles/rails_5.0.0.gemfile
+      gemfile: gemfiles/rails_5.0.0.1.gemfile
+    - rvm: 2.3.1
```

Comment:
Sorry, my confusion is because the latest 2.2 Ruby is 2.2.6 and the latest 2.3 Ruby is 2.3.3. I was wondering why we were using older ones.